public/app/config/channels.js

```diff
@@ -71,4 +71,5 @@ module.exports = {
   GET_SPACE_IN_CLASSROOM_CHANNEL: 'classroom:space:get',
   LOAD_SPACE_IN_CLASSROOM_CHANNEL: 'classroom:space:load',
   GET_SPACE_TO_LOAD_IN_CLASSROOM_CHANNEL: 'classroom:space:load:get-space',
+  TOUR_COMPLETED_CHANNEL: 'tour:complete',
```

Comment:
I think `COMPLETE_TOUR_CHANNEL` would be better.

Reply:
> * m

> Thank you for your PR! It was more complicated than I thought. I let you some comments, and here some general comment:
> 
> * Do you think having a different components for each tour would be possible? At some point your `Tour` component will have to be expanded with a lot more tours and having different components (or something else) for each might help maintaining them.
> * I used the convention to name the selectors depending on the usage (id, class, name, etc). For example I would use `VISIT_SPACE_BUTTON_CLASS` instead of `VISIT_SPACE_BUTTON` (it mostly helps for tests)

Hey Kim! Thanks!

- Ok so for using a tour component for each tour: this was the approach I was using first. Then realized that it would make the process of switching between tours much more complex. Here we are managing one reducer for all the actions related to this tour and the tours expand for more than one component. (i.e if there is a tour for example only for the space screen without having to jump between screens then maybe we could consider separating tours). Also with the current structuring, any new tour (which I assume won't be many) we just need simply add a steps array and handle the callbacks. All managed in one space. of course, I plan maybe on expanding that architecture to a more refactored one especially in the callback where it looks a bit complicated maybe in future PRs and we can utilize it in other projects.
- I will fix that